.github/workflows/pr-realm-js.yml

```diff
@@ -182,10 +182,48 @@ jobs:
           path: |
             dependencies.list
             ${{ matrix.variant.artifact-path }}
+  cluster-prep:
+    name: Prepare Clusters for Integration Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        variant:
+          - { key: linux-node }
+          - { key: linux-electron-main }
+          - { key: linux-electron-render }
+          #- { key: windows-node }
+          #- { key: windows-electron-main }
+          #- { key: windows-electron-render }
+          - { key: darwin-electron-main }
+          - { key: darwin-electron-render }
+          - { key: darwin-node }
+          - { key: android-rn }
+          - { key: ios-rn }
+          #- { key: ios-rn-catalyst }
+    steps:
+    - name: Deploy Cluster for ${{ matrix.variant.key }}
+      uses: realm/ci-actions/mdb-realm/deployApps@fac1d6958f03d71de743305ce3ab27594efbe7b7
+      id: deploy-mdb-apps
+      with:
+        realmUrl: ${{ secrets.REALM_QA_BASE_URL }}
+        projectId: ${{ secrets.ATLAS_QA_PROJECT_ID }}
+        atlasUrl: ${{ secrets.ATLAS_QA_BASE_URL}}
+        apiKey: ${{ secrets.ATLAS_QA_PUBLIC_API_KEY }}
+        privateApiKey: ${{ secrets.ATLAS_QA_PRIVATE_API_KEY }}
+        differentiator: ${{ matrix.variant.key }}-${{ github.run_id }}
+    - name: Save Cluster Name to File
+      run: |
+          echo ${{ steps.deploy-mdb-apps.outputs.clusterName }} > cluster-name.txt
```

Comment:
Do you know where `cluster-name.txt` is located on the file system?

Reply:
I forgot the `txt` in the `cat` command below.  That will probably fix it.
I'm guessing this just lands in the `github.workspace`.  Should be pretty stable, but we will see.